### PR TITLE
Create README.md

### LINKED ISSUE #1455: Create Readme.md for Hashing
@jainaman224 can you please assign this to me?

### PR DESCRIPTION
**Pull request by Chaitanyasuma**
Fixes #1455 

#### Checklist:

- [x] 4 space indentation.
- [ ] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Add README.md for Hashing

#### Languages Used:
- markdown

#### Files Added:
- README.md

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by Chaitanyasuma** [comment]
@jainaman224 Hello Sir, could you please check the PR

File: Hashing/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+2. If the index into the hash table pointed to by the hash address is not occupied then hash the key in that position.
+3. If it is occupied, then find next appropriate position using either Double Hashing or Linear Probing or Quadratic Probing
+
+## PSEUDOCODE
+
+```
+Input - hash function hf, hash table ht, key k
+
+Hashing(hf, ht) {
+  flag=0
```
Comment (SakshiShreya) [inline Hashing/README.md]: Use 4 space indentation for code.

Comment (Chaitanyasuma) [reply Hashing/README.md]: Hello Ma'am what changes should I make to the PR template

Comment (Chaitanyasuma) [reply Hashing/README.md]: @SakshiShreya Hello Ma'am, I've made the changes, please do check and let me know

**Comment by rishabhgarg25699** [comment]
@Chaitanyasuma Please squash your commit .

**Comment by rishabhgarg25699** [comment]
@Chaitanyasuma Try to squash your commit .

**Comment by MastersAbh** [comment]
Fix your PR template.

**Comment by Chaitanyasuma** [comment]
> @Chaitanyasuma Try to squash your commit .

@rishabhgarg25699 done Sir

**Comment by marwahmanbir** [comment]
@Chaitanyasuma your PR shows 16 changed files. Please remove the files that are not related to this PR, asap.

**Comment by jainaman224** [comment]
Please Fix this PR.

**Pull request closed without merging**